src/lib/tests/createMockNetworkLayer/__tests__/createMockNetworkLayer-tests.tsx

```diff
@@ -0,0 +1,87 @@
+import { IResolvers } from "graphql-tools/dist/Interfaces"
+import { graphql } from "react-relay"
+import { Environment, fetchQuery, GraphQLTaggedNode, RecordSource, Store } from "relay-runtime"
+import { createMockNetworkLayer2 } from "../index"
+
+describe("createMockNetworkLayer", () => {
```

Comment:
Could we make sure to add a comment to any files that were copied over from Reaction? It'll make it easier once we publish a package with the mock renderer stuff that both Reaction and Emission can move to 👍

Reply:
Yup, definitely will add a link to those